### ISSUE TITLE
[Bug]: (Prénom :, NOM :) is not parsed as expected from the repos' README.md

### ISSUE THREAD

**Issue by erikmd** [git4school/git4school-visu#129]
### What browsers are you seeing the problem on?

Firefox

### Short description

I tried translating the README.md template to French, but it doesn't work (see below for details).

Hopefully this will be easy to fix! Cc @F0urchette @raclet 

### Steps to reproduce

Committing in the student `main` branch this `README.md`:
```
# UPS - Licence Info - ILU1

### Prénom : John
### NOM : DOE
### Groupe de TP :
- [ ] A11
- [x] A12
- [ ] A21
- [ ] A22
- [ ] A31
- [ ] A32
- [ ] A41
- [ ] A42
- [ ] A51
- [ ] A52
- [ ] B11
- [ ] B12
```

the `Groupe de TP :` is successfully parsed by Git4School, unlike the (`Prénom :`, `NOM :`).

### Actual result

`The student's name was not found in the ReadMe file`

### Expected result

Getting `DOE John` in the `/overview` page.

This should be the expected behavior according to <https://git4school.github.io/docs/conventions.html>:

> Note that the name extraction is dependent on the selected language ! For french, the token are `NOM :` and `Prénom :`.

**Comment by erikmd** [comment]
On second thought and reading again the mentioned documentation link, it appears the issue does not occur if we select `fr` in the top right-hand-side selector **before** adding the repo for the first time.

![2022-09-19_03-23-14_Screenshot_solution](https://user-images.githubusercontent.com/10367254/190936974-1f1dddb2-5148-41ab-bdaf-8b325cc90333.png)

So I guess you can close this issue, which was a side-effect of the fact my browser is by default in en_US.

**Comment by F0urchette** [comment]
This is a usability issue that will be resolved once #122 is implemented.
